ToyBox/ReadMe.md

```diff
@@ -74,7 +74,7 @@
   skip quests you don't want to do. Be warned this may break your game progression if used carelessly.
 
 ### Ver 1.5.4 Rogue (Coming Soon)
-**ToyBox Rogue 1.5.4** - ***Sh0dan Preview*** **June 1, 2023f** for beta 0.2.1ad
+**ToyBox 1.5.4 Rogue -** ***Sh0dan*** **Preview Version 1.5.3i June 3, 2023** for beta 0.2.1af
```

Comment:
Can you please stick with the current naming convention as Rogue is part of the mod's name